File: Heat/Models/Address.vb
Imports System.ComponentModel.DataAnnotations

Namespace Models

    Public Class Address
        <Key> _
        Property ID As Integer
        Property AddressTypeID As Integer
        Property AddressType As AddressType
        Property Street As String
        Property StreetNumber As String
        Property City As String
        Property PostalCode As String
        Property Province As String
        Property State As String
        Property Phone As String
        Property CellPhone As String
        Property Fax As String
        Property Note As String

        Property CustomerID As Nullable(Of Integer)
        Overridable Property Customer As Customer


    End Class
End Namespace

File: Heat.ConvertedToC#/Models/Address.cs
using System.ComponentModel.DataAnnotations;

namespace Heat.Models
{

    public class Address
	{
		[Key()]
		public int ID { get; set; }
		public int AddressTypeID { get; set; }
		public AddressType AddressType { get; set; }
		public string Street { get; set; }
		public string StreetNumber { get; set; }
		public string City { get; set; }
		public string PostalCode { get; set; }
		public string District { get; set; }
		public string State { get; set; }
		public string Note { get; set; }

		//Property CustomerID As Nullable(Of Integer)
		//Overridable Property Customer As Customer


	}
}
